src/main/kotlin/no/nav/yrkesskade/saksbehandling/util/FristFerdigstillelseTimeManager.kt

```diff
@@ -0,0 +1,39 @@
+package no.nav.yrkesskade.saksbehandling.util
+
+import no.bekk.bekkopen.date.NorwegianDateUtil
+import no.nav.yrkesskade.saksbehandling.model.Behandlingstype
+import java.time.Instant
+import java.time.LocalDate
+import java.time.LocalDateTime
+import java.time.ZoneId
+import java.time.temporal.ChronoUnit
+import java.util.*
+
+/**
+ * Hjelpeklasse som beregner frist for behandling av en oppgave ut fra en gitt dato.
```

Comment:
Kanskje oppdatere denne javadocen litt

Reply:
Absolutt! glemte det i farten for det gir jo ingen kompileringsfeil...